internal/credentials/credentials.go

```diff
@@ -98,13 +100,14 @@ func parseCredentials(r io.Reader) (Credentials, error) {
 }
 
 func (c Credentials) write() error {
-	Debug.Print("Writing credentials: ", c)
+	util.Debug.Print("Writing credentials: ", c)
 	path := c.Filename
-	if !filepath.IsAbs(path) {
-		path = filepath.Join(CFG.FsRoot, defaulCredentialsDir, path)
-	}
+	// FIXME : Make sure this idn't breaking anything
+	// if !filepath.IsAbs(path) {
+	// 	path = filepath.Join(fsRoot, defaulCredentialsDir, path)
+	// }
```

Comment:
Can we test it in the scope of this PR? 😄

Reply:
This is a remainder for us to test this properly, since we want to move to absolute paths everywhere. We spent quiet some time to manually test this and looking through code for potential regressions where a relative path might sneak in but we weren't able to find one.

Sadly there is no unit test for these type of errors, or?